initial_setup/common.py

```diff
@@ -147,5 +147,9 @@ def get_quit_message():
 class LicensingCategory(SpokeCategory):
     displayOnHubGUI = "ProgressHub"
     displayOnHubTUI = "SummaryHub"
```

Comment:
You can probably remove these attributes. I don't think we still support them.

Reply:
Might as well do that. :)